roles/edpm_neutron_sriov/defaults/main.yml

```diff
@@ -0,0 +1,71 @@
+---
+# Copyright 2023 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+
+# All variables within this role should have a prefix of "edpm_neutron_sriov_agent"
+edpm_neutron_sriov_config_dir: "/var/lib/config-data/ansible-generated/neutron/sriov_agent"
+edpm_neutron_sriov_log_dir: "/var/log/containers/neutron"
+edpm_neutron_sriov_image: "quay.io/podified-antelope-centos9/openstack-neutron-sriov-agent:current-podified"
+
+edpm_neutron_sriov_common_volumes:
+  - /lib/modules:/lib/modules:ro
```

Comment:
(Wondering) Why does it need /lib/modules? Does it load kernel modules?

Reply:
same here

Reply:
this is already also used in container in 17.1 so I'm not sure I want/should change it now

Reply:
To clarify, I haven't requested to remove, just wondering. We may revisit the list later.